evaluation.py

```diff
@@ -38,16 +39,16 @@ def __init__(self, model_sub_dir, epoch, model_base_dir=DEFAULT_MODEL_BASE_DIR,
     # Vertically stack images of different widths
     # https://www.geeksforgeeks.org/concatenate-images-using-opencv-in-python/
     def _vstack_images(self, img_list, interpolation=cv2.INTER_CUBIC):
-        # take minimum width 
-        w_min = min(img.shape[1] for img in img_list) 
-        
-        # resizing images 
-        im_list_resize = [cv2.resize(img, 
-                        (w_min, int(img.shape[0] * w_min / img.shape[1])), 
-                                    interpolation = interpolation) 
-                        for img in img_list] 
-        # return final image 
-        return cv2.vconcat(im_list_resize) 
+        # take minimum width
+        w_min = min(img.shape[1] for img in img_list)
+
+        # resizing images
+        im_list_resize = [cv2.resize(img,
+                        (w_min, int(img.shape[0] * w_min / img.shape[1])),
+                                    interpolation = interpolation)
+                        for img in img_list]
```

Comment:
I know this isn't your change, but the formatting for this is a bit strange.

Reply:
I tried to clean it up :)